gate_fusion/BoundingBox box
gate_fusion/Keypoint[] corners
